src/input.service.ts

```diff
@@ -263,6 +263,9 @@ function isForForm(direction: Direction, selected: Element): boolean {
 @Injectable()
 export class InputService {
 
+  public inputPane : Windows.UI.ViewManagement.InputPane = Windows.UI.ViewManagement.InputPane.getForCurrentView();
+  public keyboardVisible : boolean = false;
```

Comment:
If this is going to be public, it may be more useful to have it be a `BehaviorSubject`. E.g:

```
public keyboardVisible = new BehaviorSubject(false); // initial value

alert(this.keyboardVisible.getValue()); // you can get the current value with .getValue()
this.keyboardVisible.next(true); // push updates just like an EventEmitter or rxjs Subject
```